src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -526,6 +674,424 @@ public function theAdministratorShouldSeeTheOrderWithTotalInOrderList(string $to
+                'amount' => $this->getTotalAsInt($discountedUnitPrice),
+            ],
+        );
+    }
+
+    /**
+     * @Then /^its subtotal should be ([^"]+)$/
+     */
+    public function itemSubtotalShouldBe(string $subtotal): void
+    {
+        $orderItem = $this->sharedStorage->get('item');
+
+        $unitPromotionAdjustments = 0;
+        foreach ($this->responseChecker->getCollection($this->client->getLastResponse()) as $item) {
+            if (in_array($item['type'], [AdjustmentInterface::ORDER_UNIT_PROMOTION_ADJUSTMENT, AdjustmentInterface::ORDER_PROMOTION_ADJUSTMENT])) {
+                $unitPromotionAdjustments += $item['amount'];
+            }
+        }
+
+        Assert::same($this->getTotalAsInt($subtotal), $orderItem['unitPrice'] * $orderItem['quantity'] + $unitPromotionAdjustments);
+    }
+
+    /**
+     * @Then /^its discount should be ([^"]+)$/
+     */
+    public function theItemShouldHaveDiscount(string $discount): void
+    {
+        $this->responseChecker->hasItemWithValues(
+            $this->client->getLastResponse(),
+            [
+                'type' => AdjustmentInterface::ORDER_UNIT_PROMOTION_ADJUSTMENT,
+                'amount' => $this->getTotalAsInt($discount),
+            ],
+        );
+    }
+
+    /**
+     * @Then /^its tax should be ([^"]+)$/
+     */
+    public function itemTaxShouldBe(string $tax): void
+    {
+        $this->responseChecker->hasItemWithValues(
+            $this->client->getLastResponse(),
+            [
+                'type' => AdjustmentInterface::TAX_ADJUSTMENT,
+                'amount' => $this->getTotalAsInt($tax),
+            ],
+        );
+    }
+
+    /**
+     * @Then /^its tax included in price should be ([^"]+)$/
+     */
+    public function itsTaxIncludedInPriceShouldBe(string $tax): void
+    {
+        $orderToken = $this->sharedStorage->get('order')->getTokenValue();
+        $response = $this->client->subResourceIndex(
+            Resources::ORDERS,
+            Resources::ADJUSTMENTS,
+            (string) $orderToken
+        );
+        $unitPromotionAdjustments = $this->responseChecker->getCollectionItemsWithValue(
+            $response,
+            'type',
+            AdjustmentInterface::TAX_ADJUSTMENT
+        );
+        $totalTax = 0;
+
+        foreach ($unitPromotionAdjustments as $unitPromotionAdjustment) {
+            if (true === $unitPromotionAdjustment['neutral']) {
+                $totalTax += $unitPromotionAdjustment['amount'];
+            }
+        }
+
+        Assert::same($this->getTotalAsInt($tax), $totalTax);
+    }
+
+    /**
+     * @Then I should be informed that there are no payments
+     */
+    public function iShouldSeeInformationAboutNoPayments(): void
+    {
+        Assert::same(
+            $this->responseChecker->getValue($this->client->getLastResponse(), 'payments'),
+            [],
+        );
+    }
+
+    /**
+     * @Then /^the order "[^"]+" should have order shipping state "([^"]+)"$/
```

Comment:
This step definition does not fit the arguments, as the order number will be taken as the state

Reply:
It won't be as the order number is not between parentheses.